pkg/crc/machine/fakemachine/client.go

```diff
@@ -18,8 +19,15 @@ func NewFailingClient() *Client {
 	}
 }
 
+func NewClientWithLogFile(logfile *os.File) *Client {
```

Comment:
Why a logFile here? Even in the real implementation we don't pass the logFile.

Reply:
@guillaumerose so that we can close it after delete execute, in real implementation we have `cleanup` which do the same but there the logfile path is set by the root init method.

Reply:
Is it really important to close it in `delete`? Could it be done in the test case, or is deleting a file with an open file descriptor going to be an issue in Windows?